File: API/Entities/Astronomer.cs
using System;

namespace API.Entities;

public class Astronomer
{
    public int? Id { get; set; }
    public required string Name { get; set; }
    public DateOnly DateOfBirth { get; set; }
    public required string BirthPlace { get; set; }
    public string? ImageUrl { get; set; }
    public AstronomerOccupation Occupation { get; set; }
    public string? Description { get; set; }
    public bool Married { get; set; }
    public required string Gender { get; set; }
    public int TelescopeAmount { get; set; }
}

public enum AstronomerOccupation
{
    Scientist,
    Hobbyist,
    Astronaut,
    Researcher
}

File: ClientWebsite/src/types/Astronomer.ts
export type Astronomer = {
  id?: number | null;
  name: string;
  dateOfBirth: string;
  birthPlace: string;
  imageUrl: string | null;
  occupation: AstronomerOccupation;
  description: string;
  married: boolean;
  gender: string;
  telescopeAmount: number;
}

export enum AstronomerOccupation
{
    Scientist,
    Hobbyist,
    Astronaut,
    Researcher
}
